src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,45 @@
+@use '../../styles/core/boilerplate.scss'as *;
+
+.backdrop {
+  &_wrapper {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100vh;
```

Comment:
We should test this on mobile to ensure the browser interface popping up doesn't affect view height :)